pro/src/components/VenueForm/VenueForm.tsx

```diff
@@ -81,7 +95,13 @@ const VenueForm = ({
         <Accessibility isCreatingVenue={isCreatingVenue} />
         <WithdrawalDetails isCreatedEntity={isCreatingVenue} />
         <Contact />
-        <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />
+        {canOffererCreateCollectiveOffer && isCreatingVenue
+          ? isSiretValued && (
+              <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />
+            )
+          : canOffererCreateCollectiveOffer && (
+              <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />
+            )}
```

Comment:
(style) je trouve ca un peu dur à lire, le rendu du composant est le même donc on se demande si y a vraiment une utilisé à la ternaire
En + j'ai l'impression que la condition est uniquement portée sur `canOffererCreateCollectiveOffer` ? On risque d'afficher 2 fois le composant non ? 

Du coup on pourrait simplifier avec 
```suggestion
        {canOffererCreateCollectiveOffer && <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />}
```

Reply:
jai fait la double ternaire pour evite d'avoir 
`canOffererCreateCollectiveOffer && isCreatingVenue && isSiretValued && <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />`
et 
`canOffererCreateCollectiveOffer && **!**isCreatingVenue && <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />`
la condition change en creation / edition

Reply:
en creation => canOffererCreateCollectiveOffer  && isSiretValued
en edition => canOffererCreateCollectiveOffer

Reply:
```suggestion
        {canOffererCreateCollectiveOffer && ((isCreatingVenue && isSiretValued) || !isCreatingVenue) && (
              <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />
            )
```